[debug] [2020-05-22T18:10:16.078Z] ----------------------------------------------------------------------
[debug] [2020-05-22T18:10:16.080Z] Command:       /home/mau/.nvm/versions/node/v10.19.0/bin/node /usr/bin/firebase deploy
[debug] [2020-05-22T18:10:16.081Z] CLI Version:   8.4.0
[debug] [2020-05-22T18:10:16.081Z] Platform:      linux
[debug] [2020-05-22T18:10:16.081Z] Node Version:  v10.19.0
[debug] [2020-05-22T18:10:16.082Z] Time:          Fri May 22 2020 13:10:16 GMT-0500 (Central Daylight Time)
[debug] [2020-05-22T18:10:16.082Z] ----------------------------------------------------------------------
[debug] [2020-05-22T18:10:16.082Z] 
[debug] [2020-05-22T18:10:16.091Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2020-05-22T18:10:16.091Z] > authorizing via signed-in user
[debug] [2020-05-22T18:10:16.092Z] [iam] checking project pet-adoption-buap for permissions ["firebase.projects.get","firebasehosting.sites.update"]
[debug] [2020-05-22T18:10:16.095Z] > refreshing access token with scopes: ["email","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","openid"]
[debug] [2020-05-22T18:10:16.096Z] >>> HTTP REQUEST POST https://www.googleapis.com/oauth2/v3/token  
 <request body omitted>
[debug] [2020-05-22T18:10:16.427Z] <<< HTTP RESPONSE 200 {"content-type":"application/json; charset=utf-8","vary":"X-Origin, Referer, Origin,Accept-Encoding","date":"Fri, 22 May 2020 18:10:18 GMT","server":"scaffolding on HTTPServer2","cache-control":"private","x-xss-protection":"0","x-frame-options":"SAMEORIGIN","x-content-type-options":"nosniff","alt-svc":"h3-27=\":443\"; ma=2592000,h3-25=\":443\"; ma=2592000,h3-T050=\":443\"; ma=2592000,h3-Q050=\":443\"; ma=2592000,h3-Q049=\":443\"; ma=2592000,h3-Q048=\":443\"; ma=2592000,h3-Q046=\":443\"; ma=2592000,h3-Q043=\":443\"; ma=2592000,quic=\":443\"; ma=2592000; v=\"46,43\"","accept-ranges":"none","transfer-encoding":"chunked"}
[debug] [2020-05-22T18:10:16.468Z] >>> HTTP REQUEST POST https://cloudresourcemanager.googleapis.com/v1/projects/pet-adoption-buap:testIamPermissions  
 {"permissions":["firebase.projects.get","firebasehosting.sites.update"]}
[debug] [2020-05-22T18:10:17.049Z] <<< HTTP RESPONSE 200 {"content-type":"application/json; charset=UTF-8","vary":"X-Origin, Referer, Origin,Accept-Encoding","date":"Fri, 22 May 2020 18:10:18 GMT","server":"ESF","cache-control":"private","x-xss-protection":"0","x-frame-options":"SAMEORIGIN","x-content-type-options":"nosniff","server-timing":"gfet4t7; dur=364","alt-svc":"h3-27=\":443\"; ma=2592000,h3-25=\":443\"; ma=2592000,h3-T050=\":443\"; ma=2592000,h3-Q050=\":443\"; ma=2592000,h3-Q049=\":443\"; ma=2592000,h3-Q048=\":443\"; ma=2592000,h3-Q046=\":443\"; ma=2592000,h3-Q043=\":443\"; ma=2592000,quic=\":443\"; ma=2592000; v=\"46,43\"","accept-ranges":"none","transfer-encoding":"chunked"}
[debug] [2020-05-22T18:10:17.051Z] [iam] error while checking permissions, command may fail: FirebaseError: Authorization failed. This account is missing the following required permissions on project pet-adoption-buap:

  firebase.projects.get
  firebasehosting.sites.update
[debug] [2020-05-22T18:10:17.052Z] >>> HTTP REQUEST GET https://firebase.googleapis.com/v1beta1/projects/pet-adoption-buap  
 
[debug] [2020-05-22T18:10:17.447Z] <<< HTTP RESPONSE 403 {"vary":"X-Origin, Referer, Origin,Accept-Encoding","content-type":"application/json; charset=UTF-8","date":"Fri, 22 May 2020 18:10:19 GMT","server":"ESF","cache-control":"private","x-xss-protection":"0","x-frame-options":"SAMEORIGIN","x-content-type-options":"nosniff","alt-svc":"h3-27=\":443\"; ma=2592000,h3-25=\":443\"; ma=2592000,h3-T050=\":443\"; ma=2592000,h3-Q050=\":443\"; ma=2592000,h3-Q049=\":443\"; ma=2592000,h3-Q048=\":443\"; ma=2592000,h3-Q046=\":443\"; ma=2592000,h3-Q043=\":443\"; ma=2592000,quic=\":443\"; ma=2592000; v=\"46,43\"","accept-ranges":"none","transfer-encoding":"chunked"}
[debug] [2020-05-22T18:10:17.447Z] <<< HTTP RESPONSE BODY {"error":{"code":403,"message":"The caller does not have permission","status":"PERMISSION_DENIED"}}
[debug] [2020-05-22T18:10:17.449Z] HTTP Error: 403, The caller does not have permission
[debug] [2020-05-22T18:10:17.451Z] FirebaseError: HTTP Error: 403, The caller does not have permission
    at module.exports (/usr/lib64/node_modules/firebase-tools/lib/responseToError.js:38:12)
    at Request._callback (/usr/lib64/node_modules/firebase-tools/lib/api.js:41:35)
    at Request.self.callback (/usr/lib64/node_modules/firebase-tools/node_modules/request/request.js:185:22)
    at Request.emit (events.js:198:13)
    at Request.EventEmitter.emit (domain.js:448:20)
    at Request.<anonymous> (/usr/lib64/node_modules/firebase-tools/node_modules/request/request.js:1154:10)
    at Request.emit (events.js:198:13)
    at Request.EventEmitter.emit (domain.js:448:20)
    at IncomingMessage.<anonymous> (/usr/lib64/node_modules/firebase-tools/node_modules/request/request.js:1076:12)
    at Object.onceWrapper (events.js:286:20)
    at IncomingMessage.emit (events.js:203:15)
    at IncomingMessage.EventEmitter.emit (domain.js:448:20)
    at endReadableNT (_stream_readable.js:1145:12)
    at process._tickCallback (internal/process/next_tick.js:63:19)
[error] 
[error] Error: Failed to get Firebase project pet-adoption-buap. Please make sure the project exists and your account has permission to access it.
